skeleton_chrome/content-script.js

```diff
@@ -26,10 +26,18 @@
   }
 
   // let ember-debug know that content script has executed
+  if (document) {
+    if (document.documentElement) {
+      if (!document.documentElement.dataset) {
+        document.documentElement.dataset = {};
+      }
+    } else {
+      document.documentElement = {};
+      document.documentElement.dataset = {};
+    }
+  }
   document.documentElement.dataset.emberExtension = 1;
```

Comment:
Is there a way to reference this issue officially? https://github.com/emberjs/ember-inspector/issues/463